src/middlewares/Cache.spec.ts

```diff
@@ -0,0 +1,117 @@
+import {Cache, ICacheMiddlewareStore} from "./Cache";
+import {FetchResponse} from "./Fetch";
+
+declare const Response: FetchResponse<string>;
+declare const localStorage: ICacheMiddlewareStore;
```

Comment:
i think no? we dont really conform to wallaby do we? the normal karma tests are MUST in both dev, build and CI scenario.